types/entry.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+type Foo struct {
```

Comment:
Yes, from the package, sure. But it was about the file and @svenwltr did just fix it :+1: